addons/website_livechat/controllers/chatbot.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteLivechatChatbotScriptController(http.Controller):
+    @http.route('/website_livechat/chatbot/<model("chatbot.script"):chatbot_script>/test_script',
```

Comment:
``chatbot/<model>/test`` maybe ? To keep routes simple and namespaced in chatbot ?